config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
@domcleal have you tested the snippet? I think it won't work since it's a module, including a module does not override the method and super can't be used, it'd have to use `alias_method_chain`. And then "goto https://github.com/theforeman/foreman/pull/2447/files#r32216487"